src/global_distrib/mod_global_distrib.erl

```diff
@@ -47,6 +48,116 @@ start(Host, Opts0) ->
 stop(Host) ->
     mod_global_distrib_utils:stop(?MODULE, Host, fun stop/0).
 
+-spec config_spec() -> mongoose_config_spec:config_section().
+config_spec() ->
+    #section{
+        items = #{<<"global_host">> => #option{type = string,
+                                               validate = domain},
+                  <<"local_host">> => #option{type = string,
+                                              validate = domain},
+                  <<"message_ttl">> => #option{type = integer,
+                                               validate = non_negative},
+                  <<"hosts_refresh_interval">> => #option{type = integer,
+                                                          validate = non_negative},
+                  <<"connections">> => connections_spec(),
```

Comment:
This should be mandatory according to the implementation. https://mongooseim.readthedocs.io/en/3.7.1/modules/mod_global_distrib/